apollo-ios-pagination/Sources/ApolloPagination/AnyGraphQLPager.swift

```diff
@@ -5,38 +5,40 @@ import Combine
 /// Type-erases a query pager, transforming data from a generic type to a specific type, often a view model or array of view models.
 public class AnyGraphQLQueryPager<Model> {
   public typealias Output = Result<(Model, UpdateSource), Error>
-  public var canLoadNext: Bool { pager.canLoadNext }
+  private let _subject: CurrentValueSubject<Output?, Never> = .init(nil)
+  public var publisher: AnyPublisher<Output, Never> { _subject.compactMap { $0 }.eraseToAnyPublisher() }
```

Comment:
This is perfect :)

Reply:
Can we add inline documentation to the `publisher` property to indicate that this is the intended access point for using the pager as a `Combine` stream?

Reply:
Ice